git-extra/aliases.sh

```diff
@@ -5,11 +5,11 @@
 alias ls='ls -F --color=auto --show-control-chars'
 alias ll='ls -l'
 
-case "$TERM" in
-xterm*)
+case "$MSYS" in
+*disable_pcon*)
```

Comment:
The trouble with this is: by default `MSYS` is _unset_, and Pseudo Console support is _still_ disabled. The new check misses that rather common scenario, and therefore will break them.

I did not find a way to reliably determine whether Pseudo Console support is enabled. We may need to patch the MSYS2 runtime to be able to tell, first.